core/src/main/java/nl/nn/adapterframework/jwt/JwtSecurityHandler.java

```diff
@@ -66,11 +73,12 @@ public void validateClaims(String requiredClaims, String exactMatchClaims) throw
 			Map<String, String> claims = StringUtil.splitToStream(exactMatchClaims)
 					.map(s -> StringUtil.split(s, "="))
 					.collect(Collectors.toMap(item -> item.get(0), item -> item.get(1)));
+
 			for (Map.Entry<String, String> entry : claims.entrySet()) {
 				String key = entry.getKey();
 				String expectedValue = entry.getValue();
-				String value = (String) claimsSet.get(key);
-				if(!value.equals(expectedValue)) {
+				Object value = claimsSet.get(key);
+				if(!expectedValue.equals(value)) { //Value may be a List, Long or String
```

Comment:
- Moeten we rekening houden met list van 1 element welke gelijk is aan de expectedValue?
- Als de claim een Long is, en onze expected value een string gelijk aan die Long value, is het dan niet ook een match? Met andere woorden, is claim `Long(1)` is gelijk aan `String("1")` ?

Reply:
Nee, dit kan allemaal genegeerd worden, het moet een exacte match zijn. Zoals beschreven in het commentaar representeerd een Long value een datum (epoch timestamp).